IOPool/Streamer/src/StreamerOutputModuleBase.cc

```diff
@@ -55,7 +56,8 @@ namespace {
 
 namespace edm {
   StreamerOutputModuleBase::StreamerOutputModuleBase(ParameterSet const& ps) :
-    OutputModule(ps),
+    one::OutputModuleBase::OutputModuleBase(ps),
```

Comment:
Not sure either, but without it compiler complains at template instantiation because tries to use default constructor:

/home/smorovic/CMSSW/CMSSW_7_6_0_pre4/src/EventFilter/Utilities/plugins/RecoEventOutputModuleForFU.h:88:41: error: no matching function for call to 'edm::one::OutputModuleBase::OutputModuleBase()'
     outBuf_(new unsigned char[1024*1024])